### DISCUSSION TITLE
Review Issues

### DISCUSSION THREAD

**Discussion by saeb-panahifar** [ChapterII/ngx-agora-sdk-ng#21] [General]
Hello and welcome to all contributors.

@maryam0066 
@alibsg 
@amirber 
@alidaneshvar 

I really happy to see you guys here and work with you. As I told you before, I added all issues. So If you have a questions or If It's not clear please tell me, we could talk about that. After reviewing we will have a meeting about milestones and assignments.

**Comment by saeb-panahifar** [comment]
Additionally, I added label guidelines here [Wiki / Board Label Description page](https://github.com/ChapterII/ngx-agora-sdk-ng/wiki/Project-Board-Label-Description) . You all could see what is the meaning of each label.

**Comment by ysy523** [comment]
HI, I'm the newbie for integrate the agora in angular. I have some questions regarding local and remote video  . How  your code to differentiate local and remote video ? This question is helpful for me . Thank You.
@help wanted
